recipes-core/initrdscripts/initramfs-framework/functions

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+# Copyright (C) 2019 Apertus Solutions, LLC
+# Licensed on MIT
+
+# Process any break kernel parameters
+check_break() {
+    local h="${1}"
+    local b="${2}"
+
+    if [ -n "${bootparam_break}" ]; then
+        for i in $(echo "${bootparam_break}" | sed "s/,/ /g"); do
+            if [ "${i}" = "${b}" ]; then
+                msg "Launching ${h} break hook"
```

Comment:
Shouldn't the message be: `msg "Launching ${h} ${b} hook"`? Currently it will only display: `Launching pre break hook` for any hook.

Reply:
Yes, i didn't notice that I missed it in testing before PR but as I was testing work that uses it yesterday I noticed that same point as well. So, yes will be pushing a new version today that fixes this.